core/src/jsMain/kotlin/external/BluetoothDevice.kt

```diff
@@ -0,0 +1,14 @@
+package com.juul.kable.external
+
+import org.w3c.dom.events.EventTarget
+
+// https://developer.mozilla.org/en-US/docs/Web/API/BluetoothDevice
```

Comment:
Nit: `/** */` style comments please

Sidenote: Love that Mozilla documents this even though they don't support it yet.